tests/foreman/ui/test_contentaccess.py

```diff
@@ -0,0 +1,242 @@
+    """Implements Content Access (Golden Ticket) tests in UI"""
+
+    @classmethod
+    def set_session_org(cls):
+        """Create an organization for tests, which will be selected
+        automatically
+
+        This method should set `session_org` to a new Org or reuse existing
+        org that has Golden ticket enabled
+        """
+        cls.session_org = entities.Organization().create()
+
+    @classmethod
+    def setUp(self):
+        """Setup must ensure the current `session_org`  has Golden Ticket
+        enabled.
+
+        SQL::
+
+            UPDATE
+                 cp_owner
+            SET
+                 content_access_mode = 'org_environment',
+                 content_access_mode_list='entitlement,org_environment'
+            WHERE account='{org.label}';
+
+        :steps:
+
+            1. Create a Product and CV for current session_org.
+            2. Add a repository pointing to a real repo which requires a
+               RedHat subscription to access.
+            3. Create Content Host and assign that gated repos to it.
+            4. Access candlepin PG database and execute SQL to activate
+               the Golden Ticket.
+            5. Sync the gated repository.
+
+        """
+        super(ContentAccessTestCase, self).setUp()
+
+    @run_only_on('sat')
+    @tier2
+    @stubbed()
+    def test_positive_list_installable_updates(self):
+        """Access content hosts and assert all updates are listed on
+        packages tab updates and not only those for attached subscriptions.
+
+        :id: 30783c91-c665-4c39-8b3b-b7456bde76f2
+
+        :steps:
+
+            1. Access Content-Host listing page.
+            2. Assert all content is listed and not only those for attached
+               subscriptions which can be done by checking existence of any
+               gated update.
+
+        :CaseAutomation: notautomated
+
+        :expectedresults: All updates are available independent
+            of subscription because Golden Ticket is enabled.
+
+        :CaseImportance: Critical
+        """
+
+    @run_only_on('sat')
+    @tier2
+    @stubbed()
+    def test_positive_list_available_packages(self):
+        """Access content hosts and assert all packages are listed on
+        installable updates and not only those for attached subscriptions.
+
+        :id: 37383e25-7b1d-433e-9e05-faaa8ec70ee8
+
+        :steps:
+
+            1. Access Content-Host Packages tab.
+            2. Assert all packages are listed and not only those for attached
+               subscriptions which can be done by checking existence of any
+               gated package.
+
+        :CaseAutomation: notautomated
+
+        :expectedresults: All packages are available independent
+            of subscription because Golden Ticket is enabled.
+
+        :CaseImportance: Critical
+        """
+
+    @run_only_on('sat')
+    @tier1
+    @stubbed()
+    def test_positive_visual_indicator_on_hosts_subscription(self):
```

Comment:
@sghai AFAIK the only ways are UI alert and using `rct` tool already covered in another test case